src/drt/src/dr/FlexDR.cpp

```diff
@@ -1662,6 +1675,14 @@ void FlexDR::searchRepair(int iter,
     MARKERDECAY = 0.99;
   if (iter == 50)
     MARKERDECAY = 0.999;
+  if (dist_on_) {
+    if ((iter % 10 == 0 && iter != 60) || iter == 3 || iter == 15) {
```

Comment:
What is this magic?

Reply:
these are the iterations where the globals change
0, 3, 10, 15, 20, 30, 40, 50

Reply:
It is brittle and assumes nobody touches a global somewhere else.  We can leave it for the short term but a more robust solution is

- Put the globals into an object with a single instance global instance and an operator==
- Have another instance of that object that is last_transmitted
- Check if the global != last_transmitted

Reply:
Another approach is to have an API that dry runs detailed routing, and returns metadata about the flow so that you can correctly orchestrate this stuff.